packages/datagateway-dataview/src/views/landing/isis/isisInvestigationLanding.component.test.tsx

```diff
@@ -453,4 +453,63 @@ describe('ISIS Investigation Landing page', () => {
       'John Smith et al; 2019: Test 1, doi_constants.publisher.name, https://doi.org/doi 1'
     );
   });
+
+  it('copies data citation to clipboard', () => {
+    // Mock the clipboard object
+    Object.assign(navigator, {
+      clipboard: {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        writeText: () => {},
+      },
+    });
+
+    jest.spyOn(navigator.clipboard, 'writeText');
```

Comment:
You can simplify this so the eslint disable is not necessary. I would do something like:
```
const testWriteText = jest.fn();
// Mock the clipboard object
Object.assign(navigator, {
  clipboard: {
    writeText: testWriteText,
  },
});
// jest.spyOn(navigator.clipboard, 'writeText'); no need to spyOn this anymore
expect(testWriteText).toHaveBeenCalledWith(
      'John Smith; 2019: Test 1, doi_constants.publisher.name, https://doi.org/doi 1'
);
```

Reply:
Thank you @sam-glendenning, this is a lot simpler than having to spy on the object itself.